docs/en/datasets/recog.md

```diff
@@ -471,3 +476,289 @@ rm IIIT-CVid.zip
 ```bash
 python tools/data/textdreog/lv_converter.py PATH/TO/lv
 ```
+
+### IMGUR
+
+- Step1: Run `download_imgur5k.py` to download images. You can merge [PR#5](https://github.com/facebookresearch/IMGUR5K-Handwriting-Dataset/pull/5) in your local repository to enable a **much faster** parallel execution of image download.
+
+  ```bash
+  mkdir imgur && cd imgur
+
+  git clone https://github.com/facebookresearch/IMGUR5K-Handwriting-Dataset.git
+
+  # Download images from imgur.com. This may take SEVERAL HOURS!
+  python ./IMGUR5K-Handwriting-Dataset/download_imgur5k.py --dataset_info_dir ./IMGUR5K-Handwriting-Dataset/dataset_info/ --output_dir ./imgs
+
+  # For annotations
+  mkdir annotations
+  mv ./IMGUR5K-Handwriting-Dataset/dataset_info/*.json annotations
+
+  rm -rf IMGUR5K-Handwriting-Dataset
+  ```
+
+- Step2: Generate `train_label.txt`, `val_label.txt` and `test_label.txt` and crop images with the following command:
+
+  ```bash
+  python tools/data/textrecog/imgur_converter.py PATH/TO/imgur
+  ```
+
+- After running the above codes, the directory structure should be as follows:
+
+  ```text
+  ├── imgur
+  │   ├── crops
+  │   ├── train_label.jsonl
+  │   ├── test_label.jsonl
+  │   └── val_label.jsonl
+  ```
+
+
+  ### FUNSD
```

Comment:
FUNSD should be put ahead IMGUR, and now there are some extra blank spaces before the title